torchgeo/datamodules/potsdam.py

```diff
@@ -37,7 +37,7 @@ def __init__(
             num_workers: The number of workers to use in all created DataLoaders
             val_split_pct: What percentage of the dataset to use as a validation set
         """
-        super().__init__()  # type: ignore[no-untyped-call]
```

Comment:
This shouldn't have been removed -- this is one of the reasons that mypy is complaining

Reply:
Yes, I fixed this..thanks!